pkg/kwok/controllers/pod_controller.go

```diff
@@ -363,6 +368,21 @@ func (c *PodController) configurePod(pod *corev1.Pod) ([]byte, error) {
 	})
 }
 
+func (c *PodController) markPodIP(pod *corev1.Pod) {
+	if c.cidrIPNet.Contains(net.ParseIP(pod.Status.PodIP)) {
+		c.ipPool.Use(pod.Status.PodIP)
+	}
+
+	nodeInfo := c.nodeInfoGetFunc(pod.Spec.NodeName)
+	if nodeInfo == nil || nodeInfo.CidrIPNet == nil || nodeInfo.IPPool == nil {
+		return
+	}
+
+	if nodeInfo.CidrIPNet.Contains(net.ParseIP(pod.Status.PodIP)) {
+		nodeInfo.IPPool.Use(pod.Status.PodIP)
```

Comment:
Seems to have missed the logic for reclaiming Pod IP

Reply:
@wzshiming fixed